content/manuals/engine/release-notes/27.md

```diff
@@ -27,6 +27,36 @@ For more information about:
 
 Release notes for Docker Engine version 27.4 releases.
```

Comment:
This  line (and the heading above) still points to 27.4 🙈

Reply:
Oops. fixing!